support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -11,6 +11,7 @@ export type LandingPageChoiceArchitectureAmountsFirstTestVariants = 'control'
   | 'productFirstSetOne'
   | 'productFirstSetTwo'
   | 'notintest';
+export type StripeElementsTestVariants = 'control' | 'stripeCardElement' | 'notintest';
```

Comment:
I'm confused why we are running this as a test -- isn't it a required change?

Reply:
Or is it to measure the impact of the required change even if we don't have another option?

Reply:
It is required, but the deadline for SCA was put back by 18 months so it's a bit less urgent now!
We still want to get this live regardless, but this is a pretty major change to a core part of the payment flow so it's important to check that there aren't any issues. A/B testing will allow is to compare acquisitions

Reply:
glad to know it's less urgent now